.github/actions/check_files/rpm_linux_agent_amd64.csv

```diff
@@ -5,6 +5,7 @@ full_filename,owner_name,group_name,mode,type,prot_permissions,size_bytes,size_e
 /var/ossec/tmp/sca-4.*-*-tmp/rocky/8/sca.files,root,wazuh,640,file,-rw-r-----,28,0.1
 /var/ossec/tmp/sca-4.*-*-tmp/almalinux/cis_alma_linux_8.yml,root,wazuh,640,file,-rw-r-----,393309,0.1
 /var/ossec/tmp/sca-4.*-*-tmp/almalinux/cis_alma_linux_9.yml,root,wazuh,640,file,-rw-r-----,361425,0.1
+/var/ossec/tmp/sca-4.*-*-tmp/almalinux/cis_alma_linux_10.yml,root,wazuh,640,file,-rw-r-----,361519,0.1
```

Comment:
In addition to the file `cis_alma_linux_10.yml`, it is also necessary to add the following two (check size of AlmaLinux default `sca.files`):
```
/var/ossec/tmp/sca-4.*-*-tmp/rocky/sca.files,root,wazuh,640,file,-rw-r-----,27,0.1
/var/ossec/tmp/sca-4.*-*-tmp/almalinux/sca.files,root,wazuh,640,file,-rw-r-----,30,0.1
```
This change has to be applied to all agent check files.